Libraries/Text/Text/RCTTextView.m

```diff
@@ -65,40 +83,7 @@ - (instancetype)initWithFrame:(CGRect)frame
 }
 
 #if TARGET_OS_OSX // [TODO(macOS GH#774)
-- (void)dealloc
-{
-  [self removeAllTextStorageLayoutManagers];
-}
 
-- (void)removeAllTextStorageLayoutManagers
-{
-  // On macOS AppKit can throw an uncaught exception
-  // (-[NSConcretePointerArray pointerAtIndex:]: attempt to access pointer at index ...)
-  // during the dealloc of NSLayoutManager.  The _textStorage and its
-  // associated NSLayoutManager dealloc later in an autorelease pool.
-  // Manually removing the layout managers from _textStorage prior to release
-  // works around this issue in AppKit.
-  NSArray<NSLayoutManager *> *managers = [[_textStorage layoutManagers] copy];
-  for (NSLayoutManager *manager in managers) {
-    [_textStorage removeLayoutManager:manager];
-  }
-}
-
-- (BOOL)canBecomeKeyView
-{
-	// RCTText should not get any keyboard focus unless its `selectable` prop is true
-	return _selectable;
-}
-
-- (void)drawFocusRingMask {
-  if ([self enableFocusRing]) {
-    NSRectFill([self bounds]);
-  }
-}
-
-- (NSRect)focusRingMaskBounds {
```

Comment:
@shwanton I see our fork still has focusRingMaskBounds. Should we keep it?

Reply:
Added this back in b08aae919d36fa87b03f1ee633e6e8148b66d8dc